### PR TITLE
Make google cloud  role OASIS-compatible

### LINKED ISSUE #1632: Standardize Google cloud role to OASIS.
This would require us to standardize the linchpin GCP role to OASIS. 
so that eventually it can be imported to ansible-galaxy.

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject
fixes #1632

### CODE REVIEW

File: linchpin/provision/roles/gcloud/molecule/docker/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: docker
+    image: centos:7
+    ## Remove 'image' above and uncomment the following to run systemd in Docker
```
Comment (14rcole) [inline linchpin/provision/roles/gcloud/molecule/docker/molecule.yml]: You can remove these comments

File: linchpin/provision/roles/gcloud/tasks/provision_gcloud_gcp_compute_network.yml [hunk 2]
```diff
@@ -2,22 +2,29 @@
...
+    auto_create_subnetworks: "{{ res_def['auto_create_subnetworks'] |\
+                              default('no') }}"
+    description: "{{ res_def['description'] |\
+                  default(gce_resource_name) }}"
     gateway_ipv4: "{{ res_def['gateway_ipv4'] | default(omit) }}"
     ipv4_range: "{{ res_def['ipv4_range'] | default(omit) }}"
     project: "{{ auth_var['output']['project_id'] | default(omit) }}"
-    scopes: "{{ res_def['scopes'] | default(['https://www.googleapis.com/auth/compute']) }}"
-    service_account_file: "{{ auth_var['path'] | default(omit) }}/{{ auth_var['params']['filename'] | default(omit) }}"
+    scopes: "{{ res_def['scopes'] |\
```
Comment (14rcole) [inline linchpin/provision/roles/gcloud/tasks/provision_gcloud_gcp_compute_network.yml]: nit: the backslash isn't necessary

**Comment by samvarankashyap** [comment]
@14rcole Needs review

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**